.github/ISSUE_TEMPLATE/release-manager.md

```diff
@@ -9,9 +9,9 @@ labels: sig/release, area/release-eng
 
 e.g., (at)example_user
 
-### [Release Manager](/release-managers.md) role
+### [Release Manager](https://git.k8s.io/sig-release/release-managers.md) role
```

Comment:
Would it feel too grammatically strange to have this be "Release Managers" throughout for consistency?